client/.src/containers/app/AppStyles.js

```diff
@@ -1,7 +1,9 @@
 import styled from 'styled-components';
 
-const AppStyles = styled.div`
+const AppStyles = {
+  app: styled.h1`
 background-color: red;
-`;
+`
```

Comment:
very cool to see someone working with styled components. I'll be interested in your thoughts about using styled components vs styling components via a css spreadsheet.

Reply:
It's going to be an adventure!